arrays_cycles_task_5.js

```diff
@@ -0,0 +1,6 @@
+// Sort the array [1,6,7,8,3,4,5,6] in descending order
+
+let arr = [1, 6, 7, 8, 3, 4, 5, 6];
```

Comment:
[1, 2, 7, 10, 21, 9, 33, 8] Try to do with this array, the sort() method will sort it like this: [1, 10, 2, 21, 33, 7, 8, 9]
Please read the documentation on how the sort() method works